src/config/konnectors.json

```diff
@@ -532,6 +532,27 @@
       "bankId": "168"
     }
   },
+  {
+    "slug": "cragr",
```

Comment:
Could the slug be a bit more human ;) ?

Reply:
Agree with @ptbrowne. In my opinion we don't have any constraint on slug names, so let's make it the more understandable as possible. 👍